internal/congestion/cubic_sender.go

```diff
@@ -317,8 +317,3 @@ func (c *cubicSender) OnConnectionMigration() {
 	c.slowstartThreshold = c.initialMaxCongestionWindow
 	c.maxCongestionWindow = c.initialMaxCongestionWindow
 }
-
-// SetSlowStartLargeReduction allows enabling the SSLR experiment
-func (c *cubicSender) SetSlowStartLargeReduction(enabled bool) {
-	c.slowStartLargeReduction = enabled
```

Comment:
We can probably remove the member & other references then?

Reply:
Oh nice, that removes another 10 LOC or so.